src/diff/index.js

```diff
@@ -258,32 +252,32 @@ function diffElementNodes(dom, newVNode, oldVNode, context, isSvg, excessDomChil
  * Invoke or update a ref, depending on whether it is a function or object ref.
  * @param {object|function} ref
  * @param {any} value
- * @param {import('../internal').Component} ancestorComponent
+ * @param {import('../internal').VNode} parentVNode
  */
-export function applyRef(ref, value, ancestorComponent) {
+export function applyRef(ref, value, parentVNode) {
 	try {
 		if (typeof ref=='function') ref(value);
 		else ref.current = value;
 	}
 	catch (e) {
-		catchErrorInComponent(e, ancestorComponent);
+		catchErrorInComponent(e, parentVNode);
 	}
 }
 
 /**
  * Unmount a virtual node from the tree and apply DOM changes
  * @param {import('../internal').VNode} vnode The virtual node to unmount
- * @param {import('../internal').Component} ancestorComponent The parent
+ * @param {import('../internal').VNode} parentVNode The parent
```

Comment:
The comment here still refers to a "parent component" rather than a vnode. Can the parent VNode be obtained via the `vnode._parent` link here instead of passing an argument?

Reply:
Thanks! I'll update the comment. Since this function is recursive, I wanted to keep `parentVNode` as a pointer to first parent that isn't being unmounted, thinking it should handle errors of children that are being unmounted (should an unmounting error boundary still handle errors for children as they are unmounted?). However, let me check what React does here... perhaps we can save some bytes?

Reply:
Hmm seems React does swallow errors if the nearest error boundary to a thrown error is being unmounted (https://codesandbox.io/s/react-throwing-unmount-1vdvn) (<- could someone double check me on this?)

I think I'll leave our behavior for now just cuz I think I'd prefer to make noise on errors (e.g. failing to unsubscribe some resource which could lead to a memory leak).

Reply:
Agree, swallowed errors is always painful to debug :+1: